app/services/foreman_templates/export_result.rb

```diff
@@ -2,7 +2,7 @@ module ForemanTemplates
   class ExportResult
     attr_reader :template, :name, :template_file, :exported, :additional_info
 
-    def initialize(template, exported = true)
+    def initialize(template, exported: true)
```

Comment:
I think either this API change breaks things, or it's the one `parse_result`:
```
Failure:
test_0008_should force update locked template on import(Minitest::Result) [/home/jenkins/workspace/foreman_templates-pull-request/label/fast/ruby/2.7/plugin/test/functional/api/v2/template_controller_test.rb:110]:
Expected response to be a <2XX: success>, but was a <500: Internal Server Error>
Response body: {
  "error": {"message":"wrong number of arguments (given 1, expected 0)"}
}
```

But the commit `Fix Style/OptionalBooleanParameter cop` breaks things. It's not a safe cop where you can simply change things, just like I said in https://github.com/theforeman/foreman-installer/pull/883#discussion_r1331251537.

Reply:
Even though boolean kw parameter looks nicer, I'd not change signatures just now to make rubocop happy.

Reply:
Do you want me to drop that changes related to `Style/OptionalBooleanParameter` cop?